controllers/controllerTopics.js

```diff
@@ -0,0 +1,12 @@
+const {selectTopicData} = require("../models/modelTopics")
+
+function getTopicData(req, res, next){
+    selectTopicData().then((data) => {
+        res.status(200).send(data)
```

Comment:
We should be sending the data back in an object with a key of topics like `{topics: data}`